Align oauth documentation for to the delegated auth mode (#1199)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/assembly_configuring-openshift-oauth.adoc

[id="configuring-openshift-oauth_{context}"]
= Configuring OpenShift OAuth

OAuth for OpenShift allows to authenticate the link:{site-baseurl}/che-7/openshift-connector-overview/[OpenShift connector plugin] in {prod-short}.

.Procedure

> * If {prod-short} is deployed in link:{site-baseurl}/che-7/running-che-locally/#deploying-multi-user-che-in-multi-user-mode[multi-user mode]

Reviewer: ```suggestion
* For {prod-short} deployed in link:{site-baseurl}/che-7/running-che-locally/#deploying-multi-user-che-in-multi-user-mode[multi-user mode]:
```

+

> To enable the openshift oauth automatically {prod-short} should be deployed with `--os-oauth` flag (see link:https://github.com/che-incubator/chectl#chectl-serverstart[chectl server:start specification]).

Reviewer: ```suggestion
To enable OpenShift OAuth automatically, {prod-short} should be deployed with the `--os-oauth` option (see link:https://github.com/che-incubator/chectl#chectl-serverstart[chectl server:start specification]).
```

> * If {prod-short} is deployed in single-user mode

Reviewer: ```suggestion
* For {prod-short} deployed in single-user mode:
```
